modules/virt-configuring-vm-real-time.adoc

```diff
@@ -49,56 +50,144 @@ spec:
         memory:
           guest: 4Gi
           hugepages:
-            pageSize: 1Gi <6>
+            pageSize: 1Gi # <6>
         terminationGracePeriodSeconds: 0
 # ...
 ----
 <1> This annotation specifies that load balancing is disabled for CPUs that are used by the container.
 <2> This annotation specifies that the CPU quota is disabled for CPUs that are used by the container.
 <3> This annotation specifies that interrupt request (IRQ) load balancing is disabled for CPUs that are used by the container.
-<4> The number of sockets inside the VM. 
+<4> The number of sockets inside the VM.
 <5> The number of cores inside the VM. This must be a value greater than or equal to `1`.
 <6> The size of the huge pages. The possible values for x86-64 architectures are `1Gi` and `2Mi`. In this example, the request is for 4 huge pages of size 1 Gi.
-
 . Apply the `VirtualMachine` manifest:
 +
 [source,terminal]
 ----
 $ oc apply -f <file_name>.yaml
 ----
-
-. Configure the guest operating system. The following example shows the configuration steps for a {op-system-base} 8 operating system:
+. Configure the guest operating system. The following example shows the configuration steps for a {op-system-base} 9 operating system:
 .. Run the following command to connect to the VM console:
```

Comment:
```suggestion
.. Run the following command to connect to the VM console as a root user:
```

Reply:
This is not required (and is a potential security risk), since the command is run on the user's laptop for example.
The commands which are executed inside the guest, do required root privilages.

Reply:
Understood. Then we can keep the wording and the command as is. I will mark the two comments related to this step as resolved.